vagrant/tutorial/bootstrap.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# This script performs the steps to build the base flocker-tutorial box until
+# the box must be rebooted.
+
+set -e
+
+yum update -y
+# Install useful yum repos
+yum install -y https://s3.amazonaws.com/archive.zfsonlinux.org/epel/zfs-release$(rpm -E %dist).noarch.rpm
+yum install -y epel-release
```

Comment:
I wonder how this ties in with @tomprince 's Centos7 installation docs / acceptance testing changes in #1207 :
 * https://github.com/ClusterHQ/flocker/pull/1207/files#diff-bd81b442286984f45ec7fe47f8aa0584R22

The `run-acceptance-tests` tool has a `--provisioner=vagrant`  option, which (I think) installs these repos too, so perhaps those tasks can be re-used here. Maybe create a followup issue.